unable to run gateway in docker or node

Using Docker Compose version v2.2.3

followed the instructions in the Readme but docker-compose comes up with this error:

gateway    | [cassandra] init done
gateway    | (node:67) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
gateway    | (Use `node --trace-warnings ...` to show where the warning was created)
gateway    | /app/node_modules/ts-node/src/index.ts:750
gateway    |     return new TSError(diagnosticText, diagnosticCodes);
gateway    |            ^
gateway    | TSError: ⨯ Unable to compile TypeScript:
gateway    | src/graphql/tx-search.ts(3,26): error TS2307: Cannot find module '../../static/mega-tagpairs.json' or its corresponding type declarations.
gateway    |
gateway    |     at createTSError (/app/node_modules/ts-node/src/index.ts:750:12)
gateway    |     at reportTSError (/app/node_modules/ts-node/src/index.ts:754:19)
gateway    |     at getOutput (/app/node_modules/ts-node/src/index.ts:941:36)
gateway    |     at Object.compile (/app/node_modules/ts-node/src/index.ts:1243:30)
gateway    |     at /app/node_modules/ts-node/src/esm.ts:282:37
gateway    |     at Generator.next (<anonymous>)
gateway    |     at /app/node_modules/ts-node/dist/esm.js:8:71
gateway    |     at new Promise (<anonymous>)
gateway    |     at __awaiter (/app/node_modules/ts-node/dist/esm.js:4:12)
gateway    |     at transformSource (/app/node_modules/ts-node/dist/esm.js:127:16) {
gateway    |   diagnosticText: "src/graphql/tx-search.ts(3,26): error TS2307: Cannot find module '../../static/mega-tagpairs.json' or its corresponding type declarations.\n",
gateway    |   diagnosticCodes: [ 2307 ]
gateway    | }
gateway    | error Command failed with exit code 1.
gateway    | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

over and over again

tried just running the non-docker using npm 6.14.4
ut it comes up with this error:
(I ran npm install first before getting to this point)

0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'start' ]
2 info using npm@6.14.4
3 info using node@v10.19.0
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle vartex@1.0.0-beta0~prestart: vartex@1.0.0-beta0
6 verbose lifecycle vartex@1.0.0-beta0~prestart: unsafe-perm in lifecycle true
7 verbose lifecycle vartex@1.0.0-beta0~prestart: PATH: /usr/share/npm/node_modules/npm-lifecycle/node-gyp-bin:/data/vartex/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
8 verbose lifecycle vartex@1.0.0-beta0~prestart: CWD: /data/vartex
9 silly lifecycle vartex@1.0.0-beta0~prestart: Args: [ '-c', 'yarn run gql && yarn run migrate:init' ]
10 silly lifecycle vartex@1.0.0-beta0~prestart: Returned: code: 1  signal: null
11 info lifecycle vartex@1.0.0-beta0~prestart: Failed to exec prestart script
12 verbose stack Error: vartex@1.0.0-beta0 prestart: `yarn run gql && yarn run migrate:init`
12 verbose stack Exit status 1
12 verbose stack     at EventEmitter.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/index.js:332:16)
12 verbose stack     at EventEmitter.emit (events.js:198:13)
12 verbose stack     at ChildProcess.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
12 verbose stack     at ChildProcess.emit (events.js:198:13)
12 verbose stack     at maybeClose (internal/child_process.js:982:16)
12 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
13 verbose pkgid vartex@1.0.0-beta0
14 verbose cwd /data/vartex
15 verbose Linux 5.11.0-1022-aws
16 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "start"
17 verbose node v10.19.0
18 verbose npm  v6.14.4
19 error code ELIFECYCLE
20 error errno 1
21 error vartex@1.0.0-beta0 prestart: `yarn run gql && yarn run migrate:init`
21 error Exit status 1
22 error Failed at the vartex@1.0.0-beta0 prestart script.
22 error This is probably not a problem with npm. There is likely additional logging output above.
23 verbose exit [ 1, true ]

--- a/.dockerignore
+++ b/.dockerignore
@@ -11,6 +11,7 @@
 !/cassandra/**
 !/.env
 !/patches
+!/static
 
 # Ignore Unnecessary Files
 **/*.log

